### DISCUSSION TITLE
[Icon Request] Compass

### DISCUSSION THREAD

**Discussion by lucaspatez** [tailwindlabs/heroicons#462] [Ideas]
If possible, may you please, add a compass icon to heroicons library?

It'd be nice to sections/pages like "explore" etc.

**Comment by bastinald** [comment]
+1

example: https://fontawesome.com/v5.15/icons/compass?style=regular

**Comment by ndri** [comment]
I added `compass` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=compass
